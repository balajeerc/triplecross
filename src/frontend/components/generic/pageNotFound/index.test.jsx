import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PageNotFound from './index';

beforeAll(() => {
  Enzyme.configure({ adapter: new Adapter() });
});

test('PageNotFound component renders correctly', () => {
  const component = Enzyme.shallow(<PageNotFound />);
  expect(component).toMatchSnapshot();
});
