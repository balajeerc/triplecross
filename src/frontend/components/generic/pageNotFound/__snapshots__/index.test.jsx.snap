// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageNotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Oooops... You got here because something is broken
        </h3>,
        <br />,
        <h4>
          Nothing to see here really. Would you like to back to
           
          <a
            href="/"
          >
            home page
          </a>
          ?
        </h4>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Oooops... You got here because something is broken",
        },
        "ref": null,
        "rendered": "Oooops... You got here because something is broken",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Nothing to see here really. Would you like to back to",
            " ",
            <a
              href="/"
            >
              home page
            </a>,
            "?",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Nothing to see here really. Would you like to back to",
          " ",
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "home page",
              "href": "/",
            },
            "ref": null,
            "rendered": "home page",
            "type": "a",
          },
          "?",
        ],
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Oooops... You got here because something is broken
          </h3>,
          <br />,
          <h4>
            Nothing to see here really. Would you like to back to
             
            <a
              href="/"
            >
              home page
            </a>
            ?
          </h4>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Oooops... You got here because something is broken",
          },
          "ref": null,
          "rendered": "Oooops... You got here because something is broken",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Nothing to see here really. Would you like to back to",
              " ",
              <a
                href="/"
              >
                home page
              </a>,
              "?",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Nothing to see here really. Would you like to back to",
            " ",
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "home page",
                "href": "/",
              },
              "ref": null,
              "rendered": "home page",
              "type": "a",
            },
            "?",
          ],
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
